recipes/volk/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "1.2.182":
```

Comment:
I don't know about CI, but there is 1.2.190 now.
https://github.com/zeux/volk/releases/tag/1.2.190